pkg/filters/filter.go

```diff
@@ -1,11 +1,98 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package filters
+package bloomfilter
 
-// Filter is an interface for filters
-type Filter interface {
-	Contains(pieceID []byte) bool
-	Add(pieceID []byte)
-	Encode() []byte
+import (
+	"encoding/binary"
+	"math"
+	"math/big"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Filter is a custom filter
+type Filter struct {
+	seed           int
+	k              int
+	bitsPerElement int
+	table          []byte
+}
+
+// NewFilter returns a new custom filter
+func NewFilter(expectedElements int, p float64) *Filter {
+	var f Filter
+	f.seed = 8 // TODO allow another seed value
```

Comment:
how it should be set by a function parameter? if that's the case why not to add it now? otherwise it isn't a TODO for me.

Reply:
it's because it was for benchmarking alone. I did not take the time to handle the case where the seed is higher (we can then reach the end of the byte array representing the piece id, and have to loop over it)

Reply:
I would add a short warning in the comment, otherwise it's very tentative to just add a new parameter to the function and set it without doing any other change.

On the other hand, if for now we don't plan to change it or making it configurable, I would remove the `TODO` and just add the mentioned warning.